src/Desktop/UIAutomation/A11yAutomation.cs

```diff
@@ -124,6 +144,36 @@ private static DesktopElement ElementFromUIAElement(IUIAutomationElement uia)
             return null;
         }
 
+        /// <summary>
+        /// Get DesktopElements from UIAElements.
+        /// </summary>
+        /// <param name="uia"></param>
+        /// <returns>An IEnumerable of <see cref="DesktopElement"/></returns>
+        private static IEnumerable<DesktopElement> ElementsFromUIAElements(IUIAutomationElementArray elementArray)
+        {
+            if (elementArray == null) throw new ArgumentNullException(nameof(elementArray));
+
+            var count = elementArray.Length;
+            if (count <= 0) return null;
```

Comment:
Suggestion: Instead of returning `null` here, can we return either the empty `List` or `Enumerable.Empty<DesktopElement>` ? Looking upstream, if we return `null` here, then we use that parameter to call Linq's `Any()` method, and I think that's going to trigger a `NullReferenceException`. We could add a guard before calling `Any()`, but since it already checks for an empty set, that's one less case to cover

Reply:
Updated